Advanced-Concepts/Configuring-SQL-Tables.md

```diff
@@ -32,12 +32,12 @@ The client configuration should look like this:
 </ClientConfiguration>
 ```
 
-Where the DataConnectionString is set to any valid SQL Server connection string. In order to use SQL Server as the store for system data, there’s now a script file MembershipTableCreate.sql in the Binaries\OrleansServer folder which establishes the necessary tables with the right schema. Make sure that all servers that will be hosting Orleans silos can reach the database and has access rights to it! We’ve tripped up a few times on this seemingly trivial concern, during our testing.
+Where the DataConnectionString is set to any valid SQL Server connection string. In order to use SQL Server as the store for system data, there’s now a script file MembershipTableCreate.sql in the Binaries\OrleansServer folder which establishes the necessary tables with the right schema. Make sure that all servers that will be hosting Orleans silos can reach the database and has access rights to it! We’ve tripped up a few times on this seemingly trivial concern during our testing.
 
 ### SQL Metrics and Statistics tables
 
-System tables can currently  only be stored in Azure table or SQL server.
-For Metrics and Statistics tables however we provide a generic support to host it in any persistent storage. THis is provided via the notion of StatisticsProvider. Application can write an arbitrary provider to store statistics and metrcis tables in any persistent storage. Orleans provides an implemention of one such provider: SQL Table Statistics Provider.
+System tables can currently only be stored in Azure table or SQL server.
+For Metrics and Statistics tables however, we provide a generic support to host it in any persistent storage. This is provided via the notion of a `StatisticsProvider`. Any application can write an arbitrary provider to store statistics and metrics data in a persistent store of their choice. Orleans provides an implemention of one such provider: SQL Table Statistics Provider.
 
 In order to use SQL server for statistics and metrics tables, one needs to adjust server-side and client-side configurations.
```

Comment:
I was browsing with a phone in a train and lost connection in the middle. Nothing to think about. :)